src/nxdoc/nuxeo-server/administration/maintenance/garbage-collecting-orphaned-blobs.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Garbage-Collecting Orphaned Blobs
+review:
+    comment: ''
+    date: '2023-06-09'
+    status: ok
+labels:
+    - http
+    - rest-api
+toc: true
+tree_item_index: 999
+---
+
+## Immediate Garbage Collection
+
+Since [LTS 2021-HF35]({{page page='nuxeo-server-release-notes-2021-35'}}) (see [NXP-31594](https://jira.nuxeo.com/browse/NXP-31594)), the Nuxeo Platform deletes orphaned blobs whenever a:
+- document is removed
+- document blob property is edited
+- document blob property is dispatched to another blob provider
+
+### Preconditions
+
+Only deployments using the **MongoDB** backend can benefit from this feature. All the following conditions must also be met:
```

Comment:
I am expecting a list of conditions after this, only the "queryBlobKeys capability" seems to be a requirement :thinking:

Reply:
ok this is referring to the first level subsections, not sure if this is easy to understand depending on the rendering.

Reply:
I could sum up here, all the conditions and detail them later below...